.shiprc

```diff
@@ -1,6 +1,7 @@
 {
   "files": {
-    "README.md": []
+    "README.md": [],
+    "build.gradle": ["version[[:blank:]]*=[[:blank:]]*{MAJOR}.{MINOR}.{PATCH}"]
```

Comment:
This is to prevent the ship CLI from updating other values in the build file (such as the baseline compare API diff version, or other dependency versions) from being inadvertently changed. It should only update the `version = X.X.X` in the `buildscript` block.